test_pull_requests

```diff
@@ -124,23 +124,20 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
-# TODO: create a :state to "content" mapping for composing new bot
-# messages
-PR_STATES = {
-  /^Evaluating/ => :evaluating,
-  /^Running/ => :running,
-  /^Waiting/ => :waiting,
-  /^SUCCESS/ => :success,
-  /^FAILURE/ => :failure,
-  /^ABORTED/ => :aborted,
-  /^UNSTABLE/ => :unstable,
-  /^NOT_BUILT/ => :not_built,
-  /^#{ACTION_NOT_MERGE}/ => :action_not_merge,
-  /^#{ACTION_NOT_TEAM}/ => :action_not_team,
-  /^#{ACTION_UNSUPPORTED_BRANCH}/ => :action_unsupported_branch,
+CONTENT_TO_STATE = {
+  "Evaluating for testing" => :evaluating,
+  "Running" => :running,
+  "Waiting" => :waiting, #damn and blast
```

Comment:
@danmcp The "Waiting" comment has like 3 variations, so I can't just do a `string1 == string1` match and I end up with this: https://github.com/openshift/test-pull-requests/pull/31/files/f6be253ba3714952ed04378e0054454993bcca3b..80713b15f82a591035e6a911aae567b9f974bddd#diff-c3e533b1b4fb4110b28514d90a379965R211